### ISSUE TITLE
FrozenWasteland plugin v0.5.11 checksum mismatch

### ISSUE THREAD

**Issue by cschol** [VCVRack/community#251]
Platform: Windows

From `vcv-plugindownloader` script:
```
[FrozenWasteland] Version 0.5.11
[FrozenWasteland] Downloading version 0.5.11...ERROR: Checksum verification failed
expected: df233bb718ff171fd9d9b120e44adcbf77924178e848816930356876df07c689
actual:   0d5e65f6e061127eb7bde4a5a117539da580c26de694acd5b9e87d291d4f2198
```

**Comment by WIZARDISHUNGRY** [comment]
Your checksum matches mine. @almostEric what's up?

cf #511

**Comment by cschol** [comment]
I think a scheduled Travis run is best suited for this. I pull the plugins daily and my `vcv-plugindownloader` script spits out a summary at the end with errors (checksum mismatch, failed to compile, etc.) and warnings (slug does not match root folder). I am not sure how else you would do it since you don't have control over the location of the plugins.

**Comment by cschol** [comment]
Issue fixed with #252.
